meta-refkit/conf/distro/include/refkit-development.inc

```diff
@@ -4,7 +4,7 @@ IMA_EVM_KEY_DIR = "${IMA_EVM_BASE}/data/debug-keys"
 # Enable local root access. Always use _append, to
 # avoid getting this change overwritten by a
 # REFKIT_IMAGE_EXTRA_FEATURES = "foo" in local.conf.
-REFKIT_IMAGE_EXTRA_FEATURES_append = " autologin"
+REFKIT_IMAGE_EXTRA_FEATURES_append = " autologin debug-tweaks"
```

Comment:
Why is this necessary for testing under qemu? Can't the approach used for other target hardware also be used here?

If that doesn't work (for example, because the image never gets installed for qemu), then how about "installing" a .ssh/authorized_keys via serial console? We have "autologin" active, so the serial console should be usable.